app/build.gradle

```diff
@@ -51,6 +51,7 @@ dependencies {
     implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version"
     implementation 'androidx.appcompat:appcompat:1.1.0'
     implementation 'androidx.core:core-ktx:1.2.0'
+    implementation 'androidx.lifecycle:lifecycle-livedata-ktx:2.2.0'
```

Comment:
What does this plugin do?

Reply:
At the very least, it adds an observe extension on LiveData which allows you to use function references e.g. `categoryList.observe(viewLifecycleOwner, ::handleCategories)`